libs/core/src/lib/facets/facet.module.ts

```diff
@@ -0,0 +1,23 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { AvatarModule } from '../avatar/avatar.module';
+import { ButtonModule } from '../button/button.module';
+import { FacetGroupComponent } from './facet-group.component';
+import { FacetContentComponent } from './content/facet-content.component';
+import { FacetComponent } from './facet/facet.component';
+import { TitleModule } from '../title/title.module';
+
+@NgModule({
+    declarations: [
+        FacetComponent,
+        FacetGroupComponent,
+        FacetContentComponent
+    ],
+    imports: [CommonModule, AvatarModule, ButtonModule, TitleModule],
```

Comment:
Do you need `ButtonModule` and `AvatarModule` there?

Reply:
not needed, removed this :)